src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="navigation__wraper">
+        <a href="#HomePage" class="navigation__logo">
+          <img src="images/logo.png" alt="logo MOYO">
+        </a>
+        <nav class="navigation__nav">
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a href="#Apple" class="navigation__link is-active">
+                Apple
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a href="#Samsung" class="navigation__link">
+                Samsung
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a href="#Mobile" class="navigation__link">
+                Смартфоны и телефоны
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a
+                href="#PC"
+                class="navigation__link"
+                data-qa="hover">
+                Ноутбуки и компьютеры
+              </a>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), the 3rd item